spm-headers/include/spm/seq_load.h

```diff
@@ -2,14 +2,40 @@
 
 #include <common.h>
 #include <spm/seqdrv.h>
+#include <spm/evtmgr.h>
+#include <spm/filemgr.h>
+
+#include <wii/gx.h>
 
 CPP_WRAPPER(spm::seq_load)
 
 USING(spm::seqdrv::SeqWork)
+USING(spm::evtmgr::EvtEntry)
+USING(spm::filemgr::FileEntry)
+USING(wii::gx::GXColor)
+
+typedef struct {
+    u32 flags;
+    u32 state;
+    FileEntry* fileselTpl;
+    s32 selectedSave;
+    u8 _10[0x2c-0x10];
+    EvtEntry* saveUpdateEvt;
+    s32 saveUpdateEvtId;
+    u8 _34[0x48-0x34];
+    GXColor bgGradientTop;
+    GXColor bgGradientBottom;
+    u8 _50[0x5c-0x50];
+} SeqLoadWork;
+SIZE_ASSERT(SeqLoadWork, 0x5c);
+
+DECOMP_STATIC(SeqLoadWork* seqLoadWp)
```

Comment:
In spm-headers the convention for static symbols is a snake case filename prefix and then symbol name (like `seq_load_sub_wp`